packages/vsx-registry/src/browser/vsx-extensions-contribution.ts

```diff
@@ -95,6 +99,12 @@ export class VSXExtensionsContribution extends AbstractViewContribution<VSXExten
             execute: () => this.installFromVSIX()
         });
 
+        commands.registerCommand(VSXExtensionsCommands.INSTALL_VSIX,
+            UriAwareCommandHandler.MonoSelect(this.selectionService, {
+                execute: fileURI => this.installVSIX(fileURI)
+            }
```

Comment:
nit: formatting

Reply:
This is, unfortunately, the formatted code. I can fix it, but next auto format will probably reformat it again...

Reply:
Oh, then never mind! I thought it just looked off but it's definitely better to keep the automated way ;-)

Reply:
After rewriting the code it formats better now.